Firestore/core/src/firebase/firestore/local/serializer.h

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Xcode tends to have problems with projects containing identical filenames.

Also, this class is called a `LocalSerializer` on the other platforms.

Taken together I suggest renaming this and the .cc to `local_serializer` and the class to `LocalSerializer`.

Reply:
The Redundant Department of Redundancy strikes again.  :(